.test-infra/jenkins/PreCommit_Pipeline.groovy

```diff
@@ -36,20 +36,45 @@ List<Object> commitArg = [string(name: 'sha1', value: "origin/pr/${ghprbPullId}/
 
 int javaBuildNum = NO_BUILD
 
+boolean testJava = true
+boolean testPython = true
+
+String commentLower = ghprbCommentBody.toLowerCase()
+
+if (!commentLower.isEmpty()) {
+    if (commentLower.endsWith('python only')) {
+        testJava = false
```

Comment:
This will not scale (e.g. for any future SDKs or other splits of the build pipeline).

Can you use some enumeration of permitted builds?

Reply:
Good point -- done.